sklearn/pipeline.py

```diff
@@ -131,6 +131,22 @@ def named_steps(self):
     def _final_estimator(self):
         return self.steps[-1][1]
 
+    def get_feature_names(self):
```

Comment:
I guess we should use `if_delegate_has_method` as below instead of the AttributeError here.